src/templates/audioPlayer.ejs

```diff
@@ -0,0 +1,8 @@
+<% if (zone.audio) { %>
+    <span class="audio-control playing">Pause Audio</span>
+    <audio autoplay>
+        <% _.each(zone.audio.files, function(file) { %>
```

Comment:
Alternate sources act as fallbacks. The browser stops at the first format it can play.